src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -457,6 +461,16 @@ private static void EmitNativeToInterp(StreamWriter w, List<PInvokeCallback> cal
         w.WriteLine("};");
     }
 
+    private Dictionary<Assembly, bool> assemblyDisableRuntimeMarshallingAttributeCache = new Dictionary<Assembly, bool>();
```

Comment:
```suggestion
    private Dictionary<Assembly, bool> _assemblyDisableRuntimeMarshallingAttributeCache = new ();
```

And move this to the top, with the other fields.